src/basic/include/net/if.h

```diff
@@ -6,5 +6,9 @@
 
 #define IF_NAMESIZE       16
 
+#ifndef __THROW
+#define __THROW
+#endif
```

Comment:
urks, i wasn't aware we also carry copies of glibc headers. do we really want to do that? they define an .so api after all, not a syscall api. The failure modes if something is absent are quite different hence... syscall apis have a nice failure mode, they return ENOSYS. But .so apis do not really have that.

Reply:
This is for musl.
With glibc, the macro is defined by glibc's features.h, and we use the macro as is in that case. There should be no problem.